### DISCUSSION TITLE
Will uWebSockets.js support WebTransports

### DISCUSSION THREAD

**Discussion by nasselk** [uNetworking/uWebSockets.js#1027] [General]
Hey, is it in your plan to add WebTrasnport (with UDP) support when it will be supported on nodejs/bunjs ?

**Comment by konserwa2** [comment]
What is the state of support for http3 in general? I saw an example with H3App but I didn't see it mentioned in the documentation. I don't think there are types for it either - typescript complains it doesn't exist.

**Comment by uNetworkingAB** [comment]
It is experimental and not of focus right now. Might become prioritized if needed
